projects/netgrif-components-core/src/lib/data-fields/string-collection-field/string-collection-default-field/abstract-string-collection-default-field.component.ts

```diff
@@ -0,0 +1,55 @@
+import {Component, ElementRef, Inject, OnInit, Optional, ViewChild} from '@angular/core';
+import {AbstractBaseDataFieldComponent} from '../../base-component/abstract-base-data-field.component';
+import {TranslateService} from '@ngx-translate/core';
+import {DATA_FIELD_PORTAL_DATA, DataFieldPortalData} from '../../models/data-field-portal-data-injection-token';
+import {StringCollectionField} from '../models/string-collection-field';
+import {ENTER, COMMA, SEMICOLON} from '@angular/cdk/keycodes';
+import {MatChipInputEvent} from '@angular/material/chips';
+
+@Component({
+    selector: 'ncc-abstract-string-collection-default-field',
+    template: '',
+})
+export abstract class AbstractStringCollectionDefaultFieldComponent extends AbstractBaseDataFieldComponent<StringCollectionField> implements OnInit {
+
+    @ViewChild('input') input: ElementRef;
+    public separatorKeysCodes: number[] = [ENTER];
+
+    protected constructor(protected _translate: TranslateService,
+                @Optional() @Inject(DATA_FIELD_PORTAL_DATA) dataFieldPortalData: DataFieldPortalData<StringCollectionField>) {
+        super(dataFieldPortalData);
+    }
+
+    ngOnInit() {
+        if (this.dataField?.component?.properties?.semicolon === 'true') {
+            this.separatorKeysCodes.push(SEMICOLON);
```

Comment:
Error in @angular/cdk/keycodes: 

On Slovak keyboard layout for macOS, `SEMICOLON` does not correspond to the expected input. Pressing ; results in no response, but pressing `ô` acts as a separator. Code adjustment is required for proper functionality of the `;` key to align with the Slovak keyboard layout on macOS.